test/e2e_node/container_list.go

```diff
@@ -39,10 +38,10 @@ var ImageRegistry = map[int]string{
 	hostExecImage: "gcr.io/google_containers/hostexec:1.2",
 	netExecImage:  "gcr.io/google_containers/netexec:1.4",
 	nginxImage:    "gcr.io/google_containers/nginx:1.7.9",
-	pauseImage:    "gcr.io/google_containers/pause-amd64:3.0",
```

Comment:
I think the e2e_node tests actually run on the node itself, so we could just use `runtime.GOARCH` here.